### PR TITLE
add shake

### PR DESCRIPTION
**Pull request by miku1958**
## Testing

- [x] Tested manually
- [x] Unit tests added
- [ ] UI tests added
- [x] Tests pass locally

Pod version update
- [x] Major 
- [ ] Minor 
- [ ] Patch

### CODE REVIEW

**Comment by miku1958** [comment]
I change to CoreMotion

File: Example/LayoutInspectorExampleTests/LayoutInspectorFacadeTests/LayoutInspectorFacadeTests.swift [hunk 1]
```diff
@@ -23,4 +23,16 @@ class LayoutInspectorFacadeTests: XCTestCase {
...
+    func testSubscriptionForShake() {
+        // given
+        let sharedInspectorUnderTest = LayoutInspector.shared
+		
+        // when, then
+        sharedInspectorUnderTest.setAutoTrigger(.none)
+		XCTAssertFalse(LayoutInspector.shared.motionManager.isAccelerometerActive)
+
+        sharedInspectorUnderTest.setAutoTrigger(.shake)
+	//this need to be tested in iOS device rather than simulator
```
Comment (isavynskyi) [inline Example/LayoutInspectorExampleTests/LayoutInspectorFacadeTests/LayoutInspectorFacadeTests.swift]: Nit: tabulation is broken here, and whitespace before "this" would be great

File: LayoutInspector/Classes/LayoutInspector.swift [hunk 2]
```diff
@@ -120,3 +129,27 @@ extension LayoutInspector: LayoutInspectorPresenterDelegate {
...
+		motionManager.accelerometerUpdateInterval = 0.1
+		motionManager.startAccelerometerUpdates(to: OperationQueue()) { [weak self] (data, error) in
+			guard let self = self else { return }
+			guard error == nil, let acceleration = data?.acceleration else {
+				self.motionManager.stopAccelerometerUpdates()
+				return
+			}
+			let accelerameter = sqrt( pow( acceleration.x , 2 ) + pow( acceleration.y , 2 )
+				+ pow( acceleration.z , 2) )
+			if accelerameter>2.3 {
```
Comment (isavynskyi) [inline LayoutInspector/Classes/LayoutInspector.swift]: @miku1958 is 2.3 experimentally found value?

Comment (miku1958) [reply LayoutInspector/Classes/LayoutInspector.swift]: yes

**Comment by miku1958** [comment]
I am glad to help

**Pull request merged**